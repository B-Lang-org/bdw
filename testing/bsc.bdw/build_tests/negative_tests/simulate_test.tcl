
# Simulates the project
handlers::simulate 

# Creates a new project
handlers::open_project project.bspec 

# Simulates the project
handlers::simulate

# Sets the compilation type
handlers::set_link_type make

# Simulates the project
handlers::simulate

# Sets the compilation type
handlers::set_link_type custom_command

# Simulates the project
handlers::simulate

# Closes the project 
handlers::close_project

# Exits the worksptation
exit
