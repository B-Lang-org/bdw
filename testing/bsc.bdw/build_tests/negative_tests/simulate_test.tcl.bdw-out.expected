There is no open project
The simulation executable does not exist
The simulation target is not specified
The custom command for simulation  is not specified
